sphinx/config.py

```diff
@@ -217,6 +217,12 @@ def convert_overrides(self, name, value):
             defvalue = self.values[name][0]
             if self.values[name][2] == Any:
                 return value
+            elif type(self.values[name][0]) is bool or self.values[name][2] == [bool]:
+                if value in ('0', 'False'):
```

Comment:
shouldn't our document mention  for bool config values one can now use `=True` or `=False` on command line?

Reply:
I found our document has already defined this behavior.
>For boolean values, use 0 or 1 as the value.
>https://www.sphinx-doc.org/en/master/man/sphinx-build.html#id2

I think handling `0` is a bugfix. And handling `False` is adding a new feature. I don't understand we should support `False` value also. So I drop it from this PR and post another PR if needed. Do you have any opinion for `False` value?

Reply:
I don't have a strong opinion for `False` value. All I know is that I spontaneously advised user to set option via `=False` then only later recalled one should use `=0`. I think for time being you made right choice to not add this new feature.

Reply:
Okay, it would be better to follow "Zen of python". So let's go without `False` value. And let's discuss again when somebody requests.

>There should be one-- and preferably only one --obvious way to do it.